### PR TITLE
feat: add problem module

### PR DESCRIPTION
**Pull request by Charm-Xin**
add problem module:

- add github actions and dependabot
- add gradle wrapper
- add document
- add main content

### CODE REVIEW

File: gradlew
Comment (unlimitedsola) [inline gradlew:1]: This file needs to be executable.

File: build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    compileOnly("org.springframework.boot:spring-boot-starter-security")
+}
+
+publishing {
+    publications {
+        create<MavenPublication>("maven") {
+            from(components["java"])
+            pom {
+                name.set("problem")
+                description.set("Problem support")
```
Comment (unlimitedsola) [inline build.gradle.kts]:
```suggestion
                description.set("Spring Boot Problem support")
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Problem
+This is a problem module. 
+It defines error handlers for Spring Security errors and some other common errors and their handlers. It brings unified error handling like Zalando Problem.
```
Comment (unlimitedsola) [inline README.md]: A nicer summary would be appreciated.

Comment (Charm-Xin) [reply README.md]: Are you sure? 😹

Comment (unlimitedsola) [reply README.md]: Mhm, why not?

**Comment by EggOxygen** [comment]
Can this be done and released within today? 👀

**Comment by unlimitedsola** [comment]
> Can this be done and released within today? 👀

You can help with the README to make it quicker xD

**Comment by Charm-Xin** [comment]
> Can this be done and released within today? 👀

okay...

**Review by unlimitedsola — APPROVED** [review]

**Pull request merged**